src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/Program.cs

```diff
@@ -31,13 +32,20 @@ private void ProcessRunOptions(string[] args)
             //  -   # factories to pool
             //  -   # channels to pool
             //  -   etc
-            TestHelpers.SetHostAndProtocol(useHttp: false, hostName: "localhost", appName: "WcfService1");
+            var hostName = Environment.GetEnvironmentVariable("hostName");
```

Comment:
Comment for future thought -- we have a class called TestProperties that allows us to pass values to tests either as Environment variables or msbuild properties during the build.  When you get back @KKhurin , let's see if we can use this class from stress tests, and we can add whatever properties you need.